### DISCUSSION TITLE
What is my master password?

### DISCUSSION THREAD

**Discussion by markshep** [dani-garcia/vaultwarden#3222] [Q&A]
I _must_ be overlooking something really obvious but I've just got a vaultwarden instance running and have absolutely no idea what the master password is or how to find it.

I used the two `docker` commands listed under "Installation" in the main [`README.md`](https://github.com/dani-garcia/vaultwarden/blob/main/README.md) file for the project.  So far, so good.  But then when I go the URL and enter my email address into the form it takes me to a second form that asks for the "master password".

At no point have I been informed what this master password is.  Furthermore there's no mention of "master password" in either the "Installation" section of the README file or the [FAQs](https://github.com/dani-garcia/vaultwarden/wiki/FAQs) in the wiki.  I tried nuking the `vw-data` directory and re-running the `docker run` command without the `-d` option so I could see the logs, but there's no mention of "master password" there either:

```
/--------------------------------------------------------------------\
|                        Starting Vaultwarden                        |
|                           Version 1.27.0                           |
|--------------------------------------------------------------------|
| This is an *unofficial* Bitwarden implementation, DO NOT use the   |
| official channels to report bugs/features, regardless of client.   |
| Send usage/configuration questions or feature requests to:         |
|   https://vaultwarden.discourse.group/                             |
| Report suspected bugs/issues in the software itself at:            |
|   https://github.com/dani-garcia/vaultwarden/issues/new            |
\--------------------------------------------------------------------/

[INFO] No .env file found.

[2023-02-03 12:44:00.220][vaultwarden][INFO] Private key created correctly.
[2023-02-03 12:44:00.220][vaultwarden][INFO] Public key created correctly.
[2023-02-03 12:44:00.321][start][INFO] Rocket has launched from http://0.0.0.0:80
```

I've also tried searching this discussions, but to no avail.  I'm finding it hard to believe that no-one else has encountered this problem so assume I'm being particularly stupid today.  Please can someone take pity on me and point me in the right direction?

**Comment by BlackDex** [comment]
There is no `master` password.
You need to create an account, and provide a password for that account.

**Comment by markshep** [comment]
If there's no master password why is it asking me for one?

![master-password](https://user-images.githubusercontent.com/490067/216628041-63182316-db5c-4428-ab3c-504cf43be55d.png)

**Reply by stefan0xC** [reply]
Because that is what your account password is called in Bitwarden: https://bitwarden.com/help/master-password/

**Reply by BlackDex** [reply]
I think it's best to first follow: https://bitwarden.com/help/create-bitwarden-account/
